seqr/views/utils/dataset_utils.py

```diff
@@ -276,7 +279,8 @@ def _get_unmatched_error(sample_keys):
         sample_id__in={sample_id for sample_id, _ in sample_project_tuples},
     )
 
-    samples = [s for s in samples if (s.sample_id, s.individual.family.project.name) in sample_project_tuples]
+    samples = [s for s in samples if (s.sample_id, s.individual.family.project.name) in sample_project_tuples and
+               sample_id_to_tissue_type[s.sample_id] == s.tissue_type]
```

Comment:
its not necessary, but you could also add an additional filter above for `tissue_type__in= sample_id_to_tissue_type.values()` to get back fewer incorrect samples

Reply:
I can add the tissue type condition to the filter to reduce incorrect samples. But we may still need the condition here because the filter above can get extra samples with different combinations of sample IDs and tissue types.

Reply:
we will still need the condition here, correct